src/Microsoft.Health.Fhir.SqlServer.Api/Controllers/SchemaController.cs

```diff
@@ -78,11 +83,15 @@ public FileContentResult SqlScript(int id)
         [HttpGet]
         [AllowAnonymous]
         [Route(KnownRoutes.Compatibility)]
-        public ActionResult Compatibility()
+        public async Task<IActionResult> Compatibility()
         {
             _logger.LogInformation("Attempting to get compatibility");
 
-            throw new NotImplementedException(Resources.CompatibilityNotImplemented);
+            int maxVersion = (int)_schemaInformation.MaximumSupportedVersion;
+            int minVersion = (int)_schemaInformation.MinimumSupportedVersion;
+            var compatibleResponse = await _mediator.GetCompatibleVersionAsync(minVersion, maxVersion, HttpContext.RequestAborted);
```

Comment:
This will vary based on the instance that you hit. I would think that it would be more helpful to use the current version in the DB along with the information we're recording in the instance table.